api/datastore/sql/migrations/migrations.go

```diff
@@ -55,6 +55,35 @@ var initialTables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 );`,
 }
 
+func checkOldMigrationTableVersionIfExists(db *sql.DB) (version int64, dirty bool, err error) {
+	migrationsTable := "schema_migrations"
```

Comment:
can you document this a little better please? `// "schema_migrations" is the table used by github.com/mattes/migrate to maintain migration versions in an old version of fn` -- in 3-6 months we can probably do away with this code, it's nice to leave a note so we don't have to sift through our faulty brains / github.

Reply:
fixed